internal/services/hdinsight/schema.go

```diff
@@ -978,6 +1003,8 @@ func SchemaHDInsightNodeDefinition(schemaLocation string, definition HDInsightNo
 			ForceNew:     true,
 			ValidateFunc: azure.ValidateResourceIDOrEmpty,
 		},
+
+		"script_actions": SchemaHDInsightsScriptActions(),
```

Comment:
the PR title mentions `script_action` while this block is called `scrip_actions`? it seems the title is right while the block name is incorrect?

Reply:
Hi @katbyte , the block name is correct since there could be multiple of them. Updated the PR title. Thanks.